src/restic/node.go

```diff
@@ -581,6 +581,7 @@ func (node *Node) fillExtra(path string, fi os.FileInfo) error {
 	case "file":
 		node.Size = uint64(stat.size())
 		node.Links = uint64(stat.nlink())
+		node.Device = uint64(stat.dev())
```

Comment:
I wonder if this (and the other 2 places) should be moved up, under the node.Inode =
After all, the inode is useless without the device, and if we fill one unconditionally, we should fill both.